shared/ui/components/form/inputs/DatePicker.js

```diff
@@ -244,7 +259,11 @@ DatePicker.propTypes = {
   monthFormat: PropTypes.string,
   monthYearFormat: PropTypes.string,
   weekDayFormat: PropTypes.string,
-  yearFormat: PropTypes.string
+  yearFormat: PropTypes.string,
+
+  // Positioning
```

Comment:
The fact that you made a comment here could be a sign that the prop names you chose, `horizontal` and `vertical`, might not be the most explanatory. How about `horizontalAlign` and `verticalAlign`? Too long? Maybe `halign` and `valign`? Or `hAlign` and `vAlign`?

Reply:
I made a comment here because all the other props were categorized in such a way and I was keeping it consistent. Happy to change the names to verticalAlign and horizontalAlign if that is the preference.

Reply:
Oh, that's true, I hadn't noticed that. Well, only if my comment makes sense to you.

Reply:
@e18r it looks like the Projects app uses its own DatePicker right now and not the shared one, so testing in Projects is superfluous at the moment. I'll log an issue for that shortly